src/test/shell/bazel/remote/remote_execution_test.sh

```diff
@@ -3706,4 +3706,38 @@ EOF
   expect_log "Executing genrule .* failed: (Exit 1):"
 }
 
+
+function test_local_test_execution_with_disk_cache() {
+  # Tests that the wall time for a locally executed test is correctly cached.
+  # Regression test for https://github.com/bazelbuild/bazel/issues/14426.
+
+  mkdir -p a
+  cat > a/BUILD <<EOF
+sh_test(
+  name = 'test',
+  srcs = ['test.sh'],
+)
+EOF
+  cat > a/test.sh <<EOF
+sleep 5
```

Comment:
Can we change the test to not use `sleep` as it will slow down the whole test?

Reply:
The generate-xml.sh action embeds the wall time at second granularity [1], so we must sleep for at least a second to reproduce the bug this PR fixes. I added a comment above to make this clearer.

[1] https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/exec/StandaloneTestStrategy.java;l=522;drc=1f33504af3ebe3a43aa91e607e94edc07b1807f8

Reply:
Make sense.